src/nemo_evaluator/observability/model_traffic.py

```
@@ -132,6 +133,52 @@ def _truncate(value: Any, limit: int) -> Any:
+                out["error_code"] = str(error["code"])
+        elif error is not None:
+            out["error_message"] = _truncate(str(error), limit)
+
+        if "error_message" not in out:
+            message = _first_str(body, "message", "detail", "error_message")
+            if message:
+                out["error_message"] = _truncate(message, limit)
+        try:
+            out["error_body"] = _truncate(json.dumps(body, sort_keys=True, default=str), limit)
+        except (TypeError, ValueError):
+            pass
```

Ruff (0.15.18)

[warning] Use `contextlib.suppress(TypeError, ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(TypeError, ValueError): ...`

(SIM105)